base/codevalidation.jl

```diff
@@ -0,0 +1,166 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+# Expr head => argument count bounds
+const VALID_EXPR_HEADS = Pair{Symbol,UnitRange{Int}}[
```

Comment:
It originally was, before I realized that `Dict` isn't defined yet. `Core.Inference` is a strange, strange world 😛